ingenialink/canopen/network.py

```diff
@@ -422,17 +427,21 @@ def _reset_connection(self) -> None:
                 self._connection.bus.flush_tx_buffer()
                 logger.info("Bus flushed")
         except Exception as e:
-            logger.error("Could not stop guarding. Exception: %", e)
-
+            logger.error(f"Could not stop guarding. Exception: {e}")
+        connection_args = {
+            "bustype": self.__device,
+            "channel": self.__channel,
+            "bitrate": self.__baudrate,
+        }
+        if self.__device == CAN_DEVICE.IXXAT.value:
+            connection_args["fd"] = True
         try:
-            self._connection.connect(
-                bustype=self.__device, channel=self.__channel, bitrate=self.__baudrate
-            )
+            self._connection.connect(**connection_args)
             for servo in self.servos:
                 servo.node = self._connection.add_node(servo.target)
                 servo.node.nmt.start_node_guarding(1)
```

Comment:
The node guarding period must be a constant, I know it is from another issue, but it must be fixed

Reply:
Done